tests/PHPStan/Rules/Exceptions/CatchWithUnthrownExceptionRuleTest.php

```diff
@@ -105,6 +105,10 @@ public function testRule(): void
 				'Dead catch - Exception is never thrown in the try block.',
 				398,
 			],
+			[
+				'Dead catch - Exception is never thrown in the try block.',
+				407,
```

Comment:
Why did this change?

Reply:
I am not sure, without this, it have error:

```diff
There were 2 failures:

1) PHPStan\Rules\Exceptions\CatchWithUnthrownExceptionRuleTest::testRuleWithoutReportingUncheckedException
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
 375: Dead catch - Exception is never thrown in the try block.
 380: Dead catch - Exception is never thrown in the try block.
 398: Dead catch - Exception is never thrown in the try block.
+407: Dead catch - Exception is never thrown in the try block.
```

Reply:
Ok, add namespace seems fix it https://github.com/phpstan/phpstan-src/pull/3745/commits/66596b45eb61df295e3db477ae99e4d45a48edff